lib/protocol/collection/item_collection.ts

```diff
@@ -117,23 +117,35 @@ export class ItemCollection extends MutableCollection<SNItem> {
       /** Filtered content type map */
       const filteredCTMap = this.filteredMap[contentType]!;
       const sortedElements = this.sortedMap[contentType]!;
+
+      const previousIndex = filteredCTMap[element.uuid];
+      const previousElement = !isNullOrUndefined(previousIndex) ? sortedElements[previousIndex] : undefined;
+      if (element.errorDecrypting || previousElement?.errorDecrypting) {
+        if (!element.errorDecrypting && previousElement?.errorDecrypting) {
```

Comment:
What if element is errorDecrypting, but new element is not errorDecrypting? Wouldn't we need to resort as well? In that case should this be something `element.errorDecrypting !== previousElement?.errorDecrypting`? (You'd need to account for potential of undefined on rhs).

Reply:
I did like this now. In case old value is undefined we just don't need to care about it (new element with `true` value will pass this check).